[2025-04-10T00:02:33.220+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-04-10T00:02:33.239+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: tmdb_pipeline_v2.process_kafka_data scheduled__2025-04-09T12:00:00+00:00 [queued]>
[2025-04-10T00:02:33.246+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: tmdb_pipeline_v2.process_kafka_data scheduled__2025-04-09T12:00:00+00:00 [queued]>
[2025-04-10T00:02:33.246+0000] {taskinstance.py:2303} INFO - Starting attempt 2 of 3
[2025-04-10T00:02:33.258+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): process_kafka_data> on 2025-04-09 12:00:00+00:00
[2025-04-10T00:02:33.263+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=6248) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-10T00:02:33.265+0000] {standard_task_runner.py:63} INFO - Started process 6250 to run task
[2025-04-10T00:02:33.265+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'tmdb_pipeline_v2', 'process_kafka_data', 'scheduled__2025-04-09T12:00:00+00:00', '--job-id', '998', '--raw', '--subdir', 'DAGS_FOLDER/dag_tmdb.py', '--cfg-path', '/tmp/tmp09dxr0_z']
[2025-04-10T00:02:33.268+0000] {standard_task_runner.py:91} INFO - Job 998: Subtask process_kafka_data
[2025-04-10T00:02:33.310+0000] {task_command.py:426} INFO - Running <TaskInstance: tmdb_pipeline_v2.process_kafka_data scheduled__2025-04-09T12:00:00+00:00 [running]> on host a24902f77b45
[2025-04-10T00:02:33.398+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='gabriela' AIRFLOW_CTX_DAG_ID='tmdb_pipeline_v2' AIRFLOW_CTX_TASK_ID='process_kafka_data' AIRFLOW_CTX_EXECUTION_DATE='2025-04-09T12:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-09T12:00:00+00:00'
[2025-04-10T00:02:33.399+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-04-10T00:02:33.505+0000] {dag_tmdb.py:283} INFO - Iniciando proceso de consumo de datos de Kafka
[2025-04-10T00:02:33.507+0000] {conn.py:395} INFO - <BrokerConnection client_id=kafka-python-2.1.5, node_id=bootstrap-0 host=kafka:9093 <connecting> [IPv4 ('172.19.0.6', 9093)]>: connecting to kafka:9093 [('172.19.0.6', 9093) IPv4]
[2025-04-10T00:02:33.509+0000] {conn.py:615} INFO - <BrokerConnection client_id=kafka-python-2.1.5, node_id=bootstrap-0 host=kafka:9093 <checking_api_versions_recv> [IPv4 ('172.19.0.6', 9093)]>: Broker version identified as 2.6
[2025-04-10T00:02:33.510+0000] {conn.py:456} INFO - <BrokerConnection client_id=kafka-python-2.1.5, node_id=bootstrap-0 host=kafka:9093 <connected> [IPv4 ('172.19.0.6', 9093)]>: Connection complete.
[2025-04-10T00:02:33.511+0000] {subscription_state.py:182} INFO - Updating subscribed topics to: ('tmdb_data',)
[2025-04-10T00:02:33.512+0000] {dag_tmdb.py:295} INFO - Consumidor conectado a Kafka, esperando mensajes en 'tmdb_data'...
[2025-04-10T00:02:33.512+0000] {dag_tmdb.py:339} INFO - Intento 1/3 de obtener mensajes
[2025-04-10T00:02:33.514+0000] {conn.py:395} INFO - <BrokerConnection client_id=kafka-python-2.1.5, node_id=1001 host=kafka:9093 <connecting> [IPv4 ('172.19.0.6', 9093)]>: connecting to kafka:9093 [('172.19.0.6', 9093) IPv4]
[2025-04-10T00:02:33.515+0000] {conn.py:456} INFO - <BrokerConnection client_id=kafka-python-2.1.5, node_id=1001 host=kafka:9093 <connected> [IPv4 ('172.19.0.6', 9093)]>: Connection complete.
[2025-04-10T00:02:33.515+0000] {conn.py:936} INFO - <BrokerConnection client_id=kafka-python-2.1.5, node_id=bootstrap-0 host=kafka:9093 <connected> [IPv4 ('172.19.0.6', 9093)]>: Closing connection. 
[2025-04-10T00:02:33.618+0000] {cluster.py:393} INFO - Group coordinator for ***_consumer_group_v2 is ('coordinator-1001', 'kafka', 9093, None)
[2025-04-10T00:02:33.620+0000] {base.py:729} INFO - Discovered coordinator coordinator-1001 for group ***_consumer_group_v2
[2025-04-10T00:02:33.621+0000] {base.py:780} INFO - Starting new heartbeat thread
[2025-04-10T00:02:33.622+0000] {consumer.py:361} INFO - Revoking previously assigned partitions () for group ***_consumer_group_v2
[2025-04-10T00:02:33.624+0000] {conn.py:395} INFO - <BrokerConnection client_id=kafka-python-2.1.5, node_id=coordinator-1001 host=kafka:9093 <connecting> [IPv4 ('172.19.0.6', 9093)]>: connecting to kafka:9093 [('172.19.0.6', 9093) IPv4]
[2025-04-10T00:02:33.625+0000] {conn.py:456} INFO - <BrokerConnection client_id=kafka-python-2.1.5, node_id=coordinator-1001 host=kafka:9093 <connected> [IPv4 ('172.19.0.6', 9093)]>: Connection complete.
[2025-04-10T00:02:33.727+0000] {base.py:489} INFO - (Re-)joining group ***_consumer_group_v2
[2025-04-10T00:02:33.733+0000] {base.py:552} INFO - Elected group leader -- performing partition assignments using range
[2025-04-10T00:02:33.736+0000] {base.py:380} INFO - Successfully joined group ***_consumer_group_v2 with generation 9
[2025-04-10T00:02:33.737+0000] {subscription_state.py:254} INFO - Updated partition assignment: [('tmdb_data', 0)]
[2025-04-10T00:02:33.738+0000] {consumer.py:252} INFO - Setting newly assigned partitions (('tmdb_data', 0),) for group ***_consumer_group_v2
[2025-04-10T00:02:33.786+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Parásitos
[2025-04-10T00:02:33.786+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Turno nocturno
[2025-04-10T00:02:33.787+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Giro final
[2025-04-10T00:02:33.787+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: El crepúsculo de los dioses
[2025-04-10T00:02:33.787+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Tiempos modernos
[2025-04-10T00:02:33.788+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Regreso al futuro
[2025-04-10T00:02:33.788+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Culpa mía
[2025-04-10T00:02:33.788+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Blindado
[2025-04-10T00:02:33.789+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Panda Plan
[2025-04-10T00:02:33.789+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Minecraft: The Story of Mojang
[2025-04-10T00:02:33.790+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Presence
[2025-04-10T00:02:33.790+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: La cita
[2025-04-10T00:02:33.790+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Canario Negro
[2025-04-10T00:02:33.790+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Uno de los nuestros
[2025-04-10T00:02:33.791+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Evangelion: 3.0+1.01 Thrice Upon a Time
[2025-04-10T00:02:33.791+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Mientras Cupido no está
[2025-04-10T00:02:33.791+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Policán
[2025-04-10T00:02:33.792+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: El último disparo
[2025-04-10T00:02:33.792+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Dedicada A Mi Ex
[2025-04-10T00:02:33.792+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: La venganza de Cenicienta
[2025-04-10T00:02:33.793+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Luces de la ciudad
[2025-04-10T00:02:33.793+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Radical
[2025-04-10T00:02:33.793+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: De vuelta a la acción
[2025-04-10T00:02:33.793+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Dark Match
[2025-04-10T00:02:33.794+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Ven y mira
[2025-04-10T00:02:33.794+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Érase una vez un estudio
[2025-04-10T00:02:33.794+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Milagro en la celda 7
[2025-04-10T00:02:33.795+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Seishun Buta Yarou wa Yumemiru Shoujo no Yume wo Minai
[2025-04-10T00:02:33.795+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Parthenope
[2025-04-10T00:02:33.795+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Toda una vida en un año
[2025-04-10T00:02:33.795+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Interstellar
[2025-04-10T00:02:33.796+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Sujo
[2025-04-10T00:02:33.796+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: La vida es bella
[2025-04-10T00:02:33.796+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Estado eléctrico
[2025-04-10T00:02:33.797+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: El infierno del odio
[2025-04-10T00:02:33.797+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Robot salvaje
[2025-04-10T00:02:33.797+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Wicked
[2025-04-10T00:02:33.798+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Batman Ninja vs. Yakuza League
[2025-04-10T00:02:33.798+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Forrest Gump
[2025-04-10T00:02:33.798+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Los caballeros de la mesa cuadrada y sus locos seguidores
[2025-04-10T00:02:33.798+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Los siete samuráis
[2025-04-10T00:02:33.799+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Vivir
[2025-04-10T00:02:33.799+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Henry Danger: La película
[2025-04-10T00:02:33.799+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Spermageddon
[2025-04-10T00:02:33.799+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Vengadores: Endgame
[2025-04-10T00:02:33.800+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Érase una vez en América
[2025-04-10T00:02:33.800+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Las Quintillizas: La Película
[2025-04-10T00:02:33.800+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Deadpool y Lobezno
[2025-04-10T00:02:33.800+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: The Hard Hit
[2025-04-10T00:02:33.801+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Culpa mía: Londres
[2025-04-10T00:02:33.801+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: The End of Evangelion
[2025-04-10T00:02:33.801+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Escape
[2025-04-10T00:02:33.801+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Estragos
[2025-04-10T00:02:33.802+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: La evasión
[2025-04-10T00:02:33.802+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: 825 Forest Road
[2025-04-10T00:02:33.802+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Despedidas
[2025-04-10T00:02:33.802+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Thunderbolts*
[2025-04-10T00:02:33.803+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: O Auto da Compadecida
[2025-04-10T00:02:33.803+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Blancanieves
[2025-04-10T00:02:33.803+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Until Dawn
[2025-04-10T00:02:33.803+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: El contable 2
[2025-04-10T00:02:33.804+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Josee, el tigre y los peces
[2025-04-10T00:02:33.804+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Madagascar
[2025-04-10T00:02:33.804+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Paddington: Aventura en la selva
[2025-04-10T00:02:33.804+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: El profesional (Léon)
[2025-04-10T00:02:33.805+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: La lista de Schindler
[2025-04-10T00:02:33.805+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: El Padrino Parte II
[2025-04-10T00:02:33.805+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Una pequeña confusión
[2025-04-10T00:02:33.805+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: El club de los poetas muertos
[2025-04-10T00:02:33.805+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: El señor de los anillos: La comunidad del anillo
[2025-04-10T00:02:33.806+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Death of a Unicorn
[2025-04-10T00:02:33.806+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Opus
[2025-04-10T00:02:33.806+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: En la misma clase
[2025-04-10T00:02:33.807+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Anora
[2025-04-10T00:02:33.807+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Una noche en Zoopolis
[2025-04-10T00:02:33.807+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Capitán América: Brave New World
[2025-04-10T00:02:33.807+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Ciudad de Dios
[2025-04-10T00:02:33.808+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Scream
[2025-04-10T00:02:33.808+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Doble espionaje
[2025-04-10T00:02:33.808+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Gabriel's Inferno
[2025-04-10T00:02:33.808+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Amenaza en el aire
[2025-04-10T00:02:33.809+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Pídeme lo que quieras
[2025-04-10T00:02:33.809+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: El infierno de Gabriel Parte 3
[2025-04-10T00:02:33.809+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: A Silent Voice
[2025-04-10T00:02:33.809+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Los pecadores
[2025-04-10T00:02:33.810+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: El imperio contraataca
[2025-04-10T00:02:33.810+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Orgullo y prejuicio
[2025-04-10T00:02:33.810+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Baño de Sangre
[2025-04-10T00:02:33.810+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Spider-Man: un nuevo universo
[2025-04-10T00:02:33.810+0000] {dag_tmdb.py:346} INFO - Recibido mensaje: Al otro barrio
[2025-04-10T00:02:33.811+0000] {dag_tmdb.py:349} INFO - Recibidos 90 mensajes, procediendo a procesar
[2025-04-10T00:02:33.811+0000] {dag_tmdb.py:378} INFO - Procesando 90 mensajes con Pandas
[2025-04-10T00:02:33.817+0000] {dag_tmdb.py:439} ERROR - Error en el procesamiento: sequence item 0: expected str instance, dict found
[2025-04-10T00:02:33.821+0000] {dag_tmdb.py:440} ERROR - Traceback (most recent call last):
  File "/opt/***/dags/dag_tmdb.py", line 401, in process_kafka_data
    df['cast_str'] = df['cast'].apply(lambda x: ','.join(x) if isinstance(x, list) else '')
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/***/.local/lib/python3.12/site-packages/pandas/core/series.py", line 4764, in apply
    ).apply()
      ^^^^^^^
  File "/home/***/.local/lib/python3.12/site-packages/pandas/core/apply.py", line 1209, in apply
    return self.apply_standard()
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/***/.local/lib/python3.12/site-packages/pandas/core/apply.py", line 1289, in apply_standard
    mapped = obj._map_values(
             ^^^^^^^^^^^^^^^^
  File "/home/***/.local/lib/python3.12/site-packages/pandas/core/base.py", line 921, in _map_values
    return algorithms.map_array(arr, mapper, na_action=na_action, convert=convert)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/***/.local/lib/python3.12/site-packages/pandas/core/algorithms.py", line 1814, in map_array
    return lib.map_infer(values, mapper, convert=convert)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "lib.pyx", line 2926, in pandas._libs.lib.map_infer
  File "/opt/***/dags/dag_tmdb.py", line 401, in <lambda>
    df['cast_str'] = df['cast'].apply(lambda x: ','.join(x) if isinstance(x, list) else '')
                                                ^^^^^^^^^^^
TypeError: sequence item 0: expected str instance, dict found

[2025-04-10T00:02:33.823+0000] {base.py:794} INFO - Stopping heartbeat thread
[2025-04-10T00:02:36.824+0000] {base.py:995} WARNING - Heartbeat thread did not fully terminate during close
[2025-04-10T00:02:36.826+0000] {base.py:823} INFO - Leaving consumer group (***_consumer_group_v2).
[2025-04-10T00:02:36.830+0000] {conn.py:936} INFO - <BrokerConnection client_id=kafka-python-2.1.5, node_id=1001 host=kafka:9093 <connected> [IPv4 ('172.19.0.6', 9093)]>: Closing connection. 
[2025-04-10T00:02:36.831+0000] {conn.py:936} INFO - <BrokerConnection client_id=kafka-python-2.1.5, node_id=coordinator-1001 host=kafka:9093 <connected> [IPv4 ('172.19.0.6', 9093)]>: Closing connection. 
[2025-04-10T00:02:36.832+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-04-10T00:02:36.833+0000] {taskinstance.py:2890} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/dag_tmdb.py", line 401, in process_kafka_data
    df['cast_str'] = df['cast'].apply(lambda x: ','.join(x) if isinstance(x, list) else '')
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/pandas/core/series.py", line 4764, in apply
    ).apply()
      ^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/pandas/core/apply.py", line 1209, in apply
    return self.apply_standard()
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/pandas/core/apply.py", line 1289, in apply_standard
    mapped = obj._map_values(
             ^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/pandas/core/base.py", line 921, in _map_values
    return algorithms.map_array(arr, mapper, na_action=na_action, convert=convert)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/pandas/core/algorithms.py", line 1814, in map_array
    return lib.map_infer(values, mapper, convert=convert)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "lib.pyx", line 2926, in pandas._libs.lib.map_infer
  File "/opt/airflow/dags/dag_tmdb.py", line 401, in <lambda>
    df['cast_str'] = df['cast'].apply(lambda x: ','.join(x) if isinstance(x, list) else '')
                                                ^^^^^^^^^^^
TypeError: sequence item 0: expected str instance, dict found
[2025-04-10T00:02:36.844+0000] {taskinstance.py:1205} INFO - Marking task as UP_FOR_RETRY. dag_id=tmdb_pipeline_v2, task_id=process_kafka_data, execution_date=20250409T120000, start_date=20250410T000233, end_date=20250410T000236
[2025-04-10T00:02:36.859+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 998 for task process_kafka_data (sequence item 0: expected str instance, dict found; 6250)
[2025-04-10T00:02:36.894+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2025-04-10T00:02:36.920+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-10T00:02:36.922+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
